app/krs.js

```diff
@@ -124,6 +124,13 @@ exports.provisionKey = co(function *(req) {
     throw utils.ErrorResponse(400, 'email required');
   }
 
+  const krsSpecific = req.body.krsSpecific;
```

Comment:
What does the `krsSpecific` var indicate? Let's either improve the inline + repo docs, or rename this something more meaningful.